packages/live-preview-sdk/src/messages.ts

```diff
@@ -159,3 +160,12 @@ export type MessageFromEditor = (
   from: 'live-preview';
   source: typeof LIVE_PREVIEW_EDITOR_SOURCE;
 };
+
+export function openEntryInEditorUtility(fieldId: string, entryId: string, locale: string): void {
```

Comment:
You probably need to add envId & spaceId here

Reply:
I tried it out without and it works fine🤔 I guess it just wouldn't work with x-space references, but that counts for everything in the SDK.

Reply:
It can also wait btw